exec/java-exec/pom.xml

```diff
@@ -836,6 +836,14 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>2.3.2</version>
+        <configuration>
+            <compilerArgument>-XDignore.symbol.file</compilerArgument>
```

Comment:
Consider using `io.netty.util.internal.PlatformDependent` instead of `sun.misc.Unsafe`.

Reply:
Will do!